### PR TITLE
random method switched to sample

### PR DESCRIPTION
**Pull request by Vitormdias**
Solves issue #8

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def show
 
   def random_attendee
     attendes = meetup.attendence(urlname, event_id)
-    person = attendes.shuffle.pop[:member]
+    person = attendes.sample.member
```
Comment (vitortalaia) [inline app/controllers/events_controller.rb]: What about removing this logic from the controller? Something like creating a `random_atendee` method on the service.

I also think we're gonna need to make a request to `/members/:member_id` to get a standardized profile image, as shown [here](https://www.meetup.com/pt-BR/meetup_api/docs/2/member/?uri=%2Fmeetup_api%2Fdocs%2F2%2Fmember%2F#get).

**Review by vitortalaia — CHANGES_REQUESTED** [review]

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (Vitormdias) [reply app/controllers/events_controller.rb]: @vitortalaia do you think we still need the ```attendance``` method on the service? we could just call a ```random attendee``` method from the controller

Comment (vitortalaia) [reply app/controllers/events_controller.rb]: @Vitormdias I agree, we can just call `randon_attendee` from the controller. But I think it's better to keep the `attendance` method on the service, as we can call it from within `randon_attendee` to get the participants.

Comment (Vitormdias) [reply app/controllers/events_controller.rb]: Controller:
```ruby
def random
    person = meetup.random_attendee(urlname , event_id)
    @image = meetup.profile_image(person[:id])
    @name = person[:name]
end
```
Service:
```ruby
def profile_image (member_id)
    user = parse_json ( get("/members/#{member_id}") )
    user[:photo].photo_link
end

def random_attendee(urlname , event_id)
    attendees = self.attendence(urlname , event_id)
    attendees.sample.member
end
```

what  do you think @vitortalaia ?

Comment (CaiqueMitsuoka) [reply app/controllers/events_controller.rb]: I've made a proposal in your other PR. Please look there.

File: app/services/randsome.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    def meetup
+        BaseMeetup.new @session
+    end
+
+    def hosted_by_user? (event_hosts)
+      event_hosts.any? do |host|
+        host[:id] == @session[:auth]["info"]["id"]
+      end
+    end
+end
```
Comment (vitortalaia) [inline app/services/randsome.rb]: This file indentation is different from the rest (2 spaces) :nail_care:

File: app/services/randsome.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+class Randsome
+    def initialize(session)
+        @session = session
+    end
+
+    def random_attendee(urlname , event_id)
+        attendees = meetup.attendence(urlname , event_id)
+        attendees.sample.member
```
Comment (CaiqueMitsuoka) [inline app/services/randsome.rb]: This should be `attendees.sample[:member]`. The `meetup.attendance`'s return is an array of hashs.

**Review by CaiqueMitsuoka — CHANGES_REQUESTED** [review]

**Comment by Vitormdias** [comment]
Done @CaiqueMitsuoka @vitortalaia

File: config/routes.rb [hunk 4]
```diff
@@ -5,5 +5,5 @@
   # events
   get '/events', to: 'events#index'
   get '/events/:urlname/:event_id', to: 'events#show'
-  get '/events/:urlname/:event_id/random', to: 'events#random_attendee'
+  get '/events/:urlname/:event_id/random', to: 'events#random'
```
Comment (CaiqueMitsuoka) [inline config/routes.rb]: I think this name can me discussed, neither way, to rename that route is necessary rename the template too.

Comment (CaiqueMitsuoka) [reply config/routes.rb]: I think since that method gives an attendee insted of an event, the name random_attendee is more suitable giving its context.

**Review by CaiqueMitsuoka — APPROVED** [review]

**Pull request merged**